### DISCUSSION TITLE
I can't using cerbot and apache make my instance https

### DISCUSSION THREAD

**Discussion by youshitsune** [searxng/searx-instances#95] [Q&A]
I tried optimizing my CSP and TLS grade, but I broke something. And then reinstalled whole searxng last time I made it https, but now I can't.

**Comment by tiekoetter** [comment]
@youshitsune Your certificate only includes the domain `youshitsune.me`

**Comment by youshitsune** [comment]
But how could I make certificate for searx.youshitsune.me using Let's Encrypt

**Comment by tiekoetter** [comment]
@youshitsune How did you get your current certificate?

**Comment by youshitsune** [comment]
Using certbot and Let's encrypt

**Comment by tiekoetter** [comment]
@youshitsune What command did you use? I need a bit more information to help you.

**Comment by youshitsune** [comment]
I use certbot --apache command.

**Reply by tiekoetter** [reply]
You should be able to get a certificate by running `certbot certonly -d searx.youshitsune.me`

**Reply by youshitsune** [reply]
I ran your command, but it broke my youshitsune.me website now redirects to searxng

**Reply by tiekoetter** [reply]
You probably don't have your VirtualHosts configured right.

**Reply by youshitsune** [reply]
I examined my VirtualHosts and they are configured.

**Reply by youshitsune** [reply]
Here is my searxng.conf:
```conf
# -*- coding: utf-8; mode: apache -*-

LoadModule headers_module       /usr/lib/apache2/modules/mod_headers.so
LoadModule proxy_module         /usr/lib/apache2/modules/mod_proxy.so
LoadModule proxy_uwsgi_module   /usr/lib/apache2/modules/mod_proxy_uwsgi.so
# LoadModule setenvif_module /usr/lib/apache2/modules/mod_setenvif.so

# SetEnvIf Request_URI / dontlog
# CustomLog /dev/null combined env=dontlog

<Location />

    <IfModule mod_security2.c>
        SecRuleEngine Off
    </IfModule>

    Require all granted

    Order deny,allow
    Deny from all
    # Allow from fd00::/8 192.168.0.0/16 fe80::/10 127.0.0.0/8 ::1
    Allow from all

    ProxyPreserveHost On
    ProxyPass unix:/run/uwsgi/app/searxng/socket|uwsgi://uwsgi-uds-searx/

</Location>

```

**Reply by tiekoetter** [reply]
Do you have any other way to contact you? I don't think we will get far in this github discussion.

**Reply by youshitsune** [reply]
I have my email youshitsune@protomail.com and discord

**Reply by tiekoetter** [reply]
Discord would probably work best if you don't mind.

craftsmany#6174
